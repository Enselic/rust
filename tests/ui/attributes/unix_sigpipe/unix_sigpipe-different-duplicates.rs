// revisions: with_feature without_feature

#![cfg_attr(with_feature, feature(unix_sigpipe))]

#[unix_sigpipe = "sig_dfl"]
#[unix_sigpipe = "sig_ign"] //[without_feature]~ the `#[unix_sigpipe = "sig_ign"]` attribute is an experimental feature
//~^ error: multiple `unix_sigpipe` attributes
fn main() {}
