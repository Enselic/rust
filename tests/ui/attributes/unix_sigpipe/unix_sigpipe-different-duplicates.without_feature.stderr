error[E0658]: the `#[unix_sigpipe = "sig_ign"]` attribute is an experimental feature
  --> $DIR/unix_sigpipe-different-duplicates.rs:6:1
   |
LL | #[unix_sigpipe = "sig_ign"]
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: see issue #97889 <https://github.com/rust-lang/rust/issues/97889> for more information
   = help: add `#![feature(unix_sigpipe)]` to the crate attributes to enable
   = note: this compiler was built on YYYY-MM-DD; consider upgrading it if it is out of date

error: multiple `unix_sigpipe` attributes
  --> $DIR/unix_sigpipe-different-duplicates.rs:6:1
   |
LL | #[unix_sigpipe = "sig_ign"]
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: remove this attribute
   |
note: attribute also specified here
  --> $DIR/unix_sigpipe-different-duplicates.rs:5:1
   |
LL | #[unix_sigpipe = "sig_dfl"]
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0658`.
